File: 0540-single-element-in-a-sorted-array/0540-single-element-in-a-sorted-array.py
class Solution:
    def singleNonDuplicate(self, nums: List[int]) -> int:
        ans=0
        if (len(nums)%2!=0):
            nums.append(0)
        for i in range(1,len(nums),2):
            #print(i)
            if nums[i]!=nums[i-1]:
                ans=nums[i-1]
                break
        return ans
    
            

File: 0540-single-element-in-a-sorted-array/0540-single-element-in-a-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNonDuplicate = function(nums) {
   
    if (nums.length %2 !=0)
        {
nums.push(0)
        }
  
    let j=1
    for (let i=0;j<=nums.length-1;i=i+2,j=j+2 )
        {
            if (nums[i]!=nums[j])
                {
                    return nums[i]
                }
        }
    return 0

};